calibration/CTDMOG/CGINS-CTDMOG-13435__20161015.csv

```diff
@@ -20,6 +20,6 @@ serial,name,value,notes
 37-13435,CC_ptcb1,-2.244389e-004,
 37-13435,CC_ptcb2,0.00E+00,
 37-13435,CC_ptempa0,1.717840e+002,
-37-13435,CC_ptempa1,-8.390253e-002,
+37-13435,CC_ptempa1,-8.390254e-02,
 37-13435,CC_ptempa2,4.326892e-006,
-37-13435,CC_wbotc,3.7159e-007,
+37-13435,CC_wbotc,3.715942e-07,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT